src/go/rpk/pkg/cli/cmd/cluster/config/edit/edit.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2021 Vectorized, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package edit
+
+import (
+	"io/ioutil"
+	"os"
+	"os/exec"
+
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/api/admin"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/cli/cmd/cluster/config/export"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/cli/cmd/cluster/config/importconfig"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/config"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/out"
+)
+
+func NewCommand(fs afero.Fs, all *bool) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "edit",
+		Short: "Edit cluster configuration.",
+		Long: `Edit cluster-wide configuration properties
+
+Cluster-wide properties apply to all nodes in a redpanda cluster.  For
+node properties, use the 'rpk config' command.
```

Comment:
`rpk config` is deprecated, using it prints a message that one should use `rpk redpanda config`.